### ISSUE TITLE
How to Test a Trained Model

### ISSUE THREAD

**Issue by vivid1987** [haidog-yaqub/MeanFlow#16]
May I ask how to test the performance of the trained model on the test set after training it with train.py? I particularly don't understand how to generate t and r during testing. I would greatly appreciate it if you could answer this question!

**Comment by haidog-yaqub** [comment]
> May I ask how to test the performance of the trained model on the test set after training it with train.py? I particularly don't understand how to generate t and r during testing. I would greatly appreciate it if you could answer this question!

Hi there, i implemented a very simple 1-step sampling for inference, please check this: 
```python
@torch.no_grad()
def sample_each_class(self, model, n_per_class, classes=None,
                      sample_steps=1, device='cuda'):
'''
in meanflow.py
